src/utils/safeTransactions.ts

```
@@ -18,11 +19,35 @@ import { web3 } from '@hoprnet/hopr-sdk';
 /** Human readable explanation of what the transaction is going to do */
 export const getRequestOfPendingTransaction = (transaction: SafeMultisigTransactionResponse) => {
   if (transaction.data) {
+
+    // *** Check for wrapper transactions
+    if(
+      // @ts-expect-error decode wrapper TX
+      transaction?.dataDecoded?.parameters[0]?.value === wxHOPR_WRAPPER_SMART_CONTRACT_ADDRESS &&
+      // @ts-ignore
```

ESLint

[error] Use "@ts-expect-error" instead of "@ts-ignore", as "@ts-ignore" will do nothing if the following line is error-free.

(@typescript-eslint/ban-ts-comment)